ocaml/xapi/sync_networking.ml

```diff
@@ -148,6 +148,23 @@ let copy_bonds_from_master ~__context () =
 (* This is now executed fully on the master, once asked by the slave when the slave's Xapi starts up *)
 let copy_vlans_from_master ~__context () =
   let host = !Xapi_globs.localhost_ref in
+  let master = Helpers.get_master ~__context in
```

Comment:
I don't really understand why this patch is needed. Does the pool join code not put management on the right PIF?